lib/ansible/modules/windows/win_lineinfile.ps1

```diff
@@ -95,11 +95,18 @@ function WriteLines($outlines, $path, $linesep, $encodingobj, $validate) {
 		}
 
 	}
-	
+
 	# Commit changes to the path
 	$cleanpath = $path.Replace("/", "\");
-	Copy-Item $temppath $cleanpath -force;	
+	Try {
+		Copy-Item $temppath $cleanpath -force;
+	}
+	catch {
+		Fail-Json "Cannot write to: $cleanpath"
```

Comment:
I should have been clearer... but it would be great if this returned the actual exception message as well as your generic message.  After all, there's lots of possible reasons why you might not be able to copy - this gives users a clue where to go looking to fix the problem.  (sometimes - I find some microsoft error messages to be confusing).

Something like
Fail-Json "Cannot write to: $cleanpath. (" + $_.Exception.Message + ")"